hs628_tools/Test.py

```diff
@@ -0,0 +1,133 @@
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+
+
+
+
+
+
+class data_split:
+    """
+    Create class data_split to predict a perfect train and test data split ratio.
+
+    defines method to accomplish this task.
+
+    returns a split ratio.
+
+    Identifies Missing values as Nan for any missing value, return train and test dataframe with uniform division
+    of NAN between train and test data per the data split ratio.
+
+    """
+    def __init__(self, data):
+        self.data = data
+
+
+    def test_train():
+        """
+        Reads in csv file using np.genfromtxt, counts number of NANs in the dataset
+
+        """
+
+        data = np.genfromtxt(csv_datafile, delimiter=',', skip_header=1)
+        np.isnan(data).sum() #counts the number of "NANs" in dat
+        #print data.shape
+
+
+
+        #set calculation standard for specific datalengths, this is based on the general rule of thumb
+        #ratio increases as the length of the data increases, so to ensure we have  enough data to train and test
+        #for the smaller datasets
+        
+        if len(data) <= 500:
+            train = 60
+            test = 40
+        elif len(data) > 500 and row_len <= 10000:
+            train = 70
+            test = 30
+        elif len(data) > 10000 and row_len <=100000:
+            train = 75
+            test = 25
+        else:
+            train = 80
+            test = 20
+        print 'train:',train, 'test:',test
```

Comment:
This seems like a bit of testing code that got left behind.  If you are intending it to stay then the output should be more clear (and should be mentioned in the docstring).

Reply:
@EHWUSF necessary changes per the recommendation has been made. Added "Suggests train and test ratio and prints out the train_percentage for user's convenience" to the docstring.